### PR TITLE
Migrate `DeletePatientScreen` to a fragment

### PR DESCRIPTION
**Pull request by msasikanth**
https://app.shortcut.com/simpledotorg/story/6199/migrate-deletepatientscreen-to-a-fragment

### CODE REVIEW

File: app/src/main/java/org/simple/clinic/editpatient/deletepatient/DeletePatientScreen.kt [hunk 1]
```diff
@@ -182,4 +182,9 @@ class DeletePatientScreen(
 
     override fun instantiateFragment() = DeletePatientScreen()
   }
+
+  interface Injector {
+
```
Comment (Honey14) [inline app/src/main/java/org/simple/clinic/editpatient/deletepatient/DeletePatientScreen.kt:160]: Extra space here

File: app/src/main/java/org/simple/clinic/editpatient/deletepatient/DeletePatientScreen.kt [hunk 2]
```diff
@@ -1,6 +1,7 @@
 package org.simple.clinic.editpatient.deletepatient
```
Comment (Honey14) [inline app/src/main/java/org/simple/clinic/editpatient/deletepatient/DeletePatientScreen.kt:1]: Commit message does not make sense, can you reword it better?

**Review by Honey14 — CHANGES_REQUESTED** [review]
LTGM, left 2 comments. PTAL

File: app/src/main/java/org/simple/clinic/editpatient/deletepatient/DeletePatientScreen.kt (hunk 1 above)
Comment (msasikanth) [reply app/src/main/java/org/simple/clinic/editpatient/deletepatient/DeletePatientScreen.kt:160]: That's intentional.

File: app/src/main/java/org/simple/clinic/editpatient/deletepatient/DeletePatientScreen.kt (hunk 2 above)
Comment (msasikanth) [reply app/src/main/java/org/simple/clinic/editpatient/deletepatient/DeletePatientScreen.kt:1]: I think the commit message specifies exactly what we are doing here. We are injecting into `DeletePatientScreen` in `onAttach`. 🤔

**Pull request merged**